Dynamic-Programming/tests/LevenshteinDistance.test.js

```diff
@@ -0,0 +1,19 @@
+import { calculateLevenshteinDp } from '../LevenshteinDistance'
+
+test('Add and remove letters at the beginning and the end while making replacements', () => {
```

Comment:
Suggestion :
"Should return the distance based on the characters in the middle of the strings"

Reply:
The test titles on the repository have various formats. 

@appgurueu Please let me know if you also think adding "should return the distance" at the beginning is preferable, or a general title as it is is enough.

Reply:
@JCarlosR, the importance is that the description of your tests, follow a pattern. For readability :)